django_cas_ng/models.py

```diff
@@ -17,7 +17,7 @@ class ProxyError(ValueError):
 class ProxyGrantingTicket(models.Model):
     class Meta:
         unique_together = ('session_key', 'user')
-    session_key = models.CharField(max_length=255, blank=True, null=True)
+    session_key = models.CharField(max_length=1025, blank=True, null=True)
```

Comment:
The new max length (1025) does not match the value in the migration (1024).

Reply:
woops, thanks. Fix is here: https://github.com/django-cas-ng/django-cas-ng/pull/263

Reply:
👍🏻